main.py

```diff
@@ -2,30 +2,18 @@
 import os
 import random
 import time
+import json
 
-titles = [
-    "ranting about jira",
-    "siting in a meeting",
-    "squashing bugs",
-    "centering divs",
-    "waiting for staging deployment",
-    "writing tests",
-    "pushing directly to prod",
-    "doing code review",
-    "deleting jira tickets",
-    "Creating a pull request",
-    "forgot to run DB migrations",
-    "Broke production",
-    "running on staging ENV",
-    "merging git branches",
-    "releasing the MVP",
-    "finding JIRA tickets"
-]
+with open('dataset.json', 'r') as f:
+    data = json.load(f)
+
+titles = data['titles']
```

Comment:
Please move this snippet inside `get_titles()` functions since get titles invloves reading the file as a subtask.